db/migrate/20240221211359_notification_request_ids_to_timestamp_ids.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class NotificationRequestIdsToTimestampIds < ActiveRecord::Migration[7.1]
+  def up
+    safety_assured do
+      execute("ALTER TABLE notification_requests ALTER COLUMN id SET DEFAULT timestamp_id('notification_requests')")
```

Comment:
Is it possible to create the column this way in the initial migration ... or does it need to be a follow-up alter?

Reply:
Don't think there's a way to make it part of `create_table`, but maybe it could be part of the same migration. Either way, I'm doing it the same way all previous migrations of this type have been done and don't see any downside to it.